tests/integration-tests/tests/cloudwatch_logging/test_compute_console_output_logging.py

```diff
@@ -0,0 +1,168 @@
+import json
+import logging
+import re
+
+import boto3
+import pytest
+from assertpy import assert_that
+from clusters_factory import Cluster
+from configparser import ConfigParser
+from remote_command_executor import RemoteCommandExecutor
+from retrying import retry
+from time_utils import minutes
+
+from tests.cloudwatch_logging.cloudwatch_logging_boto3_utils import (
+    get_cluster_log_groups_from_boto3,
+    get_log_events,
+    get_log_streams,
+)
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="function")
+def test_resources_dir(datadir):
+    return datadir / "resources"
+
+
+def _get_infra_stack_outputs(stack_name, region_name):
+    cfn = boto3.client("cloudformation", region_name=region_name)
+    return {
+        entry.get("OutputKey"): entry.get("OutputValue")
+        for entry in cfn.describe_stacks(StackName=stack_name)["Stacks"][0]["Outputs"]
+    }
+
+
+@retry(stop_max_attempt_number=15, wait_fixed=minutes(3))
+def _verify_compute_console_output_log_exists_in_log_group(cluster):
+    log_groups = get_cluster_log_groups_from_boto3(f"/aws/parallelcluster/{cluster.name}")
+    assert_that(log_groups).is_length(1)
+    log_group_name = log_groups[0].get("logGroupName")
+    log_streams = get_log_streams(log_group_name)
+    streams = [
+        stream.get("logStreamName")
+        for stream in log_streams
+        if re.fullmatch(r".*\.compute_console_output", stream.get("logStreamName"))
+    ]
+    assert_that(streams).is_length(1)
+    stream_name = streams[0]
+    events = get_log_events(log_group_name, stream_name)
+    messages = (event.get("message") for event in events)
+    assert_that(
+        [
+            message
+            for message in messages
+            if re.fullmatch(
+                r"2\d{3}-\d{1,2}-\d{1,2} \d{2}(:\d{2}){2},\d{3} - Console output for node compute-st-.*", message
+            )
+        ]
+    ).is_not_empty()
+
+
+@pytest.mark.usefixtures("os", "instance", "scheduler")
+def test_compute_console_logging(
```

Comment:
Is this test added to common.yaml?

Reply:
Can this test be merged with `test_monitoring_enabled_configures_console_output`?

Reply:
This is a test that I'd prefer to run manually since it takes so long to execute - basically it has to wait for for a compute node to fail to connect to DynamoDB, be detected by clustermgtd, and then wait 5 minutes to collect the console output. I left it in because I want to be able to test the exact case this is solving for (i.e. no connectivity from the compute node). However, there is another test that tries to fail faster by running a bad post-install script, and maybe I should combine this with that test.

Reply:
Makes sense. Can you add the explanation to a comment in this test? We sometimes check unrun tests. With a comment, we will be able to know why.

Reply:
Added comment, wdyt?